integreat_cms/cms/templates/hix_widget.html

```diff
@@ -49,10 +49,15 @@
                     <i icon-name="info"></i>
                     {% translate "There is no page content." %}
                 </div>
+                <div data-hix-state="loading" class="bg-blue-400 rounded p-2">
+                    <i icon-name="info"></i>
+                    {% translate "HIX widget is loading..." %}
+                </div>
```

Comment:
There is already a loading div which is working when DeepL is disabled:
https://github.com/digitalfabrik/integreat-cms/blob/c27133f3ac9ad5fe41941cb0b7ca3ab73e1f7d39/integreat_cms/cms/templates/hix_widget.html#L13-L18

Do you think you could merge them somehow, so we don't have two different loading elements?

Reply:
We could just remove the new one completely, if that's OK with you? This just ensures that a message is always shown.

Reply:
Ah sorry, now I get it, the new one is not the overlay, but just the message that is shown behind the overlay? Ok, yeah I guess it's fine removing it, since it is covered by the loading overlay anyway?

Reply:
Ok, but somehow this change caused the overlay to not be shown anymore? :sweat_smile: 
Probably it makes sense to remove the `hidden` class here:
https://github.com/digitalfabrik/integreat-cms/blob/9536578edc22bc3df607622345b03951ac65fbf0/integreat_cms/cms/templates/hix_widget.html#L14

And only hide the loading overlay after the value really has been initialized?

Reply:
Whoops, sorry. Yes.